docs/source/_static/css/custom.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
```

Comment:
Reconsider link colors on front page. Also does this mean any api changing PR in the future should also have docs changes?